src/sagemaker/serve/builder/transformers_builder.py

```diff
@@ -246,18 +255,22 @@ def _build_transformers_env(self):
 
         _create_dir_structure(self.model_path)
         if not hasattr(self, "pysdk_model"):
-            self.env_vars.update({"HF_MODEL_ID": self.model})
+
+            if self.inference_spec is not None:
+                self.env_vars.update({"HF_MODEL_ID": self.inference_spec.get_model()})
```

Comment:
This means that customer needs to implement `get_model` right? Do we need to update `InferenceSpec`

Reply:
Will make that update now, good catch